src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -4,87 +4,95 @@ short-title: Integrate Flutter
+TODO(jmagman): IDE instructions
 
-If you're starting off with an *existing* host app for either platform, you'll
-likely want to include your Flutter project in that app as some form of library
-instead.
+This creates a Flutter module project at `some/path/my_flutter/`.
 
-In order to embed Flutter as a library, use the Flutter module template.
-Executing `flutter create -t module xxx` produces a Flutter project with a
-CocoaPods pod designed for consumption by your existing host app.
+In that directory you can run the same `flutter` commands you would
+in any other Flutter project, like `flutter run --debug` or `flutter build ios`.
+You can also run in [Android Studio/IntelliJ][] or [VS Code][] with the Flutter and Dart plugins.
+This will run a single-view example version of your module without being
+embedded in your existing application. This can be useful for testing
+quick, incremental changes with hot reload.
 
-Assume you have an existing iOS app at `some/path/MyApp`, and that you
-want your Flutter project as a sibling:
+### Module organization
+The `my_flutter` module directory structure is similar to a normal Flutter
+application:
 
-```bash
-cd some/path/
-flutter create -t module my_flutter
+```text
+my_flutter/
+  .ios/
+    Runner.xcworkspace
+    Flutter/podhelper.rb
+  lib/
+    main.dart
+  test/
+  pubspec.yaml
 ```
 
-This creates a `some/path/my_flutter/` Flutter module project with some Dart
-code to get you started, and a `.ios/` hidden subfolder. The `.ios/` folder wraps
-the module project that contains some CocoaPods and a helper Ruby script.
+Add your Dart code to the `lib/` directory.
+
+You can add Flutter dependencies to `my_flutter/pubspec.yaml`, including Flutter plugins 
+with iOS platform code.
+
+There is a `.ios/` hidden subfolder which contains a Xcode workspace where you can 
+run a stand-alone version of your module. It also contains helper scripts to embed
+the module into your existing application with [CocoaPods][].
 
-## Make the host app depend on the Flutter module
+{{site.alert.note}}
+Custom iOS code should be added to your existing application or a plugin, not
+the module in `.ios/`. Changes made in `.ios/` will not be embedded in your existing application.
+The directory can be regenerated by running `flutter clean` in the `my_flutter` directory.
+{{site.alert.end}}
 
-The description below assumes that your existing iOS app has a structure similar
-to what you get by asking Xcode to generate a new "Single View App" project
-using Objective-C. If your existing app has a different folder structure
-and/or existing `.xcconfig` files, you can reuse those, but you'll probably need
-to adjust some of the relative paths mentioned below accordingly.
+## Embed the Flutter module in your existing application
 
-The assumed folder structure is as follows:
+There are two ways to embed Flutter in your existing application.
+1. Use the CocoaPods dependency manager and `flutter`. Recommended.
```

Comment:
Not clear when you just say `flutter`. I suspect you're trying to say that in this case, you need a locally installed Flutter SDK.

Reply:
I meant the `flutter` command.  I will change to Flutter SDK.